samples/vue-query/src/components/pet.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="pet" v-if="pet && petId" :key="pet.id">
+    {{ petId }}
+    {{ pet }}
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { computed, unref } from 'vue';
+import { useShowPetById } from '../api/endpoints/petstoreFromFileSpecWithTransformer';
+
+const props = defineProps<{ petId: string }>();
+const petId = computed(() => props.petId);
```

Comment:
the computed is not really need here right?

Reply:
The props is a reactive object. Wrapping in the computed can help to demonstrate reactivity of the individual prop though?

https://vuejs.org/guide/extras/reactivity-transform.html#reactive-props-destructure

Let me know if you want this changed?

Reply:
no it's was to be sure